<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>房贷计算器</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;房贷计算器&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款类别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>计算方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelfangwudanjia">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房屋单价：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelfangwumianji">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房屋面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按揭成数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按揭年数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款利率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>还款方式：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCal">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空重填</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMorType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>商业贷款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公积金贷款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>组合型贷款</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxCalType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>根据面积、单价计算</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>根据贷款总额计算</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditPrice">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditarea">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPercentage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>8成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.5成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.5成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.5成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.5成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.5成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.5成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2成</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxNumofYear">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>25年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1年</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditRate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>等额本息</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>等额本金</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>211</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>计算结果</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>首付：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="shoufu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>94</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>每月月供：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yuegong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>首月月供：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="shouyuegong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>每月递减：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="meiyuedijian">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>贷款总额：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="zonge">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>支付利息：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lixi">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>还款总额：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="huankuanzonge">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>还款月数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="huankuanyueshu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>月</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEditDaikuanzonge">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labeldaikuanzonge">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款总额：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditshangyedaikuangzonge">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditgongjijindaikuanzonge">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelshangdaizonge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>商业贷款总额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelgongjijindaizonge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>公积金贷总额：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditshangdaiRate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditgongjijinRate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelgongjijindaililv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>公积金贷利率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelshangdaililv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>商业贷款利率</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
